Jenkinsfile

```diff
@@ -140,149 +128,92 @@ pipeline {
                 cp gitlab/dev/conf/* conf
```

Comment:
119-129 - don't think these are needed anymore are we don't use synthetic csv data anymore.

Reply:
This task isn't about refactoring the pipeline entirely, I just need to remove references to branches. We've been scope creaping already - I would suggest raising a Jira if this isnt required.